.rubocop.yml

```diff
@@ -13,3 +13,12 @@ Style/DocumentationMethod:
 
 Style/ClassAndModuleChildren:
   Enabled: false
+
```

Comment:
If we want to make style changes in feedjira, we should probably discuss it in an issue first.